todoapp/app/src/main/AndroidManifest.xml

```diff
@@ -15,9 +14,9 @@
   ~ limitations under the License.
   -->
 
-<manifest
-    package="com.example.android.architecture.blueprints.todoapp"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
why is tools needed?

Reply:
@florina-muntenescu tools in xml files are used to fix any Android Lint issue like "app is not indexed by Google" or no backup data preference. Here, this attribute is used only to fix "UnusedAttribute" Android Lint issue. 
Disclaimer: I suppose you 

I would wonder rather why this issue was fixed, than `why is tools needed` :-)

Reply:
What Piotrek said.